hydrocron/db/track_ingest.py

```diff
@@ -28,44 +32,144 @@ class Track:
     newly discovered granules for Hydrocron database ingestion.
     """
 
+    BATCH_STATUS = int(os.getenv("BATCH_STATUS")) if os.getenv("BATCH_STATUS") else None
     CMR_API = "https://cmr.earthdata.nasa.gov/search/granules.umm_json"
+    COUNTER_RANGE = int(os.getenv("COUNTER_RANGE")) if os.getenv("COUNTER_RANGE") else 1
+    DEBUG_LOGS = bool(int(os.getenv("DEBUG_LOGS"))) if os.getenv("DEBUG_LOGS") else False
+    ENV = os.getenv("HYDROCRON_ENV").lower()
     PAGE_SIZE = 2000
-
-    def __init__(self, collection_shortname, collection_start_date, hydrocron_table):
+    FEATURE_ID = {
+        "SWOT_L2_HR_RiverSP_reach_2.0": "reach_id",
```

Comment:
I want to be careful about putting things in too many different places throughout the code that will need to be updated with new collection names/versions - should this be moved to the constants file? If not let's document that it will need to be updated with the table names etc for each version. Maybe this is ok as-is for now but we roll changing this into the 1.6.0 release that will have the new table/collection dictionary in the constants file. What do you think @nikki-t

Reply:
@torimcd - I agree that we should move this into the constants file so that we only have one file that needs to be changed with each new collection version and I think it makes sense to make this change a part of the 1.6.0 release since it will cover handling the new collections. Should we create a separate issue to track this? Since it is quite a small task, I can also complete this when I work on #233.

Reply:
I'm ok with rolling it into #233. Thanks!